### PR TITLE
Update New-CsOnlineAudioFile.md

### PR DESCRIPTION
**Pull request by CLYVR**
Removing information for a cmdlet that is no longer supported and refering to cmdlet to use in it's place.  Will look to remove this article entirely at end of Q1.  Missed this one when all the other ones were done.

### CODE REVIEW

**Comment by CLYVR** [comment]
@get-itips @dariomws @tseward @tiburd Please and thanks!

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/New-CsOnlineAudioFile.md [hunk 1]
```diff
@@ -13,96 +13,8 @@ ms.reviewer:
...
-
-[Get-CsOnlineAudioFile](Get-CsOnlineAudioFile.md)
-
-[Import-CsOnlineAudioFile](Import-CsOnlineAudioFile.md)
-
-[Remove-CsOnlineAudioFile](Remove-CsOnlineAudioFile.md)
+> [!CAUTION]
+> This cmdlet has been deprecated and is no longer supported.
+> 
+> Please use [Import-CsOnlineAudioFile](Import-CsOnlineAudioFile.md) cmdlet instead.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsOnlineAudioFile.md:19]:
```suggestion
> This cmdlet has been deprecated and is no longer supported. Use the [Import-CsOnlineAudioFile](Import-CsOnlineAudioFile.md) cmdlet instead.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@CLYVR Please implement the suggested changes. Thanks

**Review by get-itips — APPROVED** [review]

**Pull request merged**